lib/i18n/backend/active_record.rb

```diff
@@ -4,9 +4,24 @@
 module I18n
   module Backend
     class ActiveRecord
-      autoload :Missing,     'i18n/backend/active_record/missing'
-      autoload :StoreProcs,  'i18n/backend/active_record/store_procs'
-      autoload :Translation, 'i18n/backend/active_record/translation'
+      autoload :Missing,       'i18n/backend/active_record/missing'
+      autoload :StoreProcs,    'i18n/backend/active_record/store_procs'
+      autoload :Translation,   'i18n/backend/active_record/translation'
+      autoload :Configuration, 'i18n/backend/active_record/configuration'
+
+      class << self
+        def configure(&block)
+          yield(config) if block_given?
+        end
+
+        def config
+          @config ||= Configuration.new
+        end
+
+        def config=(x)
```

Comment:
Woah, very cool - I hadn't considered `I18n::Backend::ActiveRecord` as itself being an instance of `Class`, and thus having its own instance variables. I learned something here, thanks man 👏!

I've made the change and squashed the commits again. Let me know if there's anything else.